src/main/java/com/hotels/service/tracing/zipkintohaystack/forwarders/Fork.java

```diff
@@ -38,7 +41,7 @@ public Fork(@Autowired(required = false) SpanForwarder... spanForwarders) {
     @PostConstruct
     public void init() {
         if (spanForwarders.length == 0) {
-            throw new IllegalStateException("No span forwarders configured. See README.md for a list of available forwarders.");
+            logger.warn("No span forwarders configured. Pitchfork will not process any span.");
```

Comment:
Wont fail fast anymore after startup. I want to remove the logging forwarder and replace it with a general logging feature and this wouldn't help. I think there are valid scenarios where we to want to start the app in a no-op mode (for example testing a k8s deployment).